get-resource.sh

```diff
@@ -63,6 +63,11 @@ else
       unxz "$RHCOS_IMAGE_FILENAME_RAW"
     fi
 
+    if [ -n "$IP_OPTIONS" ] ; then
```

Comment:
This could be undefined, so it'd be safer to do `${IP_OPTIONS:-}` or else default this at the top of the script?